org.mhogomchungu.media-downloader.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "org.mhogomchungu.media_downloader",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "command": "org.mhogomchungu.media_downloader",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
it's either x11 alone or wayland + fallback-x11. not both. unless you have a good reason.